lib/presentation/journey/todo/create_todo/create_todo_screen.dart

```diff
@@ -10,24 +16,98 @@ class CreateTodoScreen extends StatefulWidget {
 }
 
 class _CreateTodoScreenState extends State<CreateTodoScreen> {
+  final TextEditingController _descController = new TextEditingController();
+  TodoBloc _todoBloc;
+
   @override
   void initState() {
+    _todoBloc = Injector.resolve<TodoBloc>();
     super.initState();
   }
 
   @override
   void dispose() {
     super.dispose();
+    _todoBloc?.close();
   }
 
   @override
-  Widget build(BuildContext buildContext) => Scaffold(
-        appBar: AppBar(
-          title: Text('Create Todo'),
-          centerTitle: true,
-        ),
-        body: Center(
-          child: Text('CODE !'),
+  Widget build(BuildContext buildContext) {
+    return Scaffold(
+      appBar: AppBar(
+        title: Text('Create Todo'),
+        centerTitle: true,
+      ),
+      body: Center(
+        child: BlocProvider<TodoBloc>(
+          create: (BuildContext context) => _todoBloc,
+          child: BlocBuilder<TodoBloc, TodoState>(
+            builder: (BuildContext context, TodoState todoState) {
+              if (todoState is TodoInitial) {
+                return Padding(
+                  padding: EdgeInsets.all(20),
+                  child: Column(
+                    children: [
+                      TextField(
+                        controller: _descController,
+                        decoration: InputDecoration(labelText: "description"),
+                      ),
+                      SizedBox(
+                        height: 20,
+                      ),
+                      RaisedButton(
+                        onPressed: () {
+                          String desc = _descController.text;
+                          _saveNewTodo(desc);
+                        },
+                        child: Text("Save"),
+                      )
+                    ],
+                  ),
+                );
+              } else if (todoState is CreatingTodo) {
+                return CircularProgressIndicator();
+              } else if (todoState is TodoCreated) {
```

Comment:
For this state, 
lets have bloc consumer and in listen, lets navigate back to todo list screen